pymodbus/framer/binary_framer.py

```diff
@@ -41,12 +41,12 @@ class ModbusBinaryFramer(ModbusFramer):
 
     method = "binary"
 
-    def __init__(self, decoder, client=None):
+    def __init__(self, decoder, **kwargs):
```

Comment:
it does not take any parameter, only client=

Reply:
see comment above